README.md

```diff
@@ -228,20 +228,111 @@ Now everything has been set up and OP-TEE is ready to be used.
 ---
 ### 4.3 Juno
 Juno has been supported in OP-TEE since mid October 2014.
-#### 4.3.1 Get the compiler and source
-Follow the instructions in the "4.1 Basic setup".
 
-#### 4.3.2 Build
+WARNING: 
+setup_juno_optee.sh script provides a coherent set of components (OP-TEE client/driver/os,
+Linux kernel version 3-16.0-rc5)
+Further release will align the ARM Juno setup with other OP-TEE supported platforms:
++ Linux kernel version alignment (3.18-rc1) with QEMU/FVP (DMA_BUF API change)
++ Will need arch/arm/Kconfig patch(es) (i.e DMA_SHARED_BUFFER etc...)
+Temporary patch files required for linux kernel and juno dtb definition.
+config.linux-linaro-tracking.a226b22057c22b433caafc58eeae6e9b13ac6c8d.patch
```

Comment:
To make it a bit more readable, could you please put the config.linux-linaro-tracking.a226b22057c22b433caafc58eeae6e9b13ac6c8d.patch under code snippets like this instead: `config.linux-linaro-tracking.a226b22057c22b433caafc58eeae6e9b13ac6c8d.patch`?

The same at next line and also `arch/arm/Kconfig ` at line 237.